jobs/e2e_node/containerd/hosts-k8s.toml

```diff
@@ -0,0 +1,7 @@
+server = "https://k8s.gcr.io"
```

Comment:
we really shouldn't need this anymore, jobs should not be pulling from k8s.gcr.io, if they still are then that's a bug.

Reply:
will remove this